umap/static/umap/js/umap.core.js

```diff
@@ -161,7 +161,7 @@ L.Util.detectFileType = (f) => {
 }
 
 L.Util.usableOption = (options, option) =>
-  options[option] !== undefined && options[option] !== ''
```

Comment:
To make `getDynamicOption()` return the default I had to exclude `null` here. I tested with the test case data from `_pre.js`. Is there any other special meaning for `null`, that should be `usable`?

Reply:
Yes, in all ternary fields, like https://github.com/umap-project/umap/blob/master/umap/static/umap/js/umap.forms.js#L796-L802